runtime/kusama/constants/src/weights/block_weights.rs

```diff
@@ -1,28 +1,27 @@
-// This file is part of Substrate.
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
```

Comment:
I added the correct file header manually. All future MRs should have them automatically https://github.com/paritytech/polkadot/issues/5984